src/memefactory/server/syncer.cljs

```diff
@@ -124,26 +129,35 @@
                        (doseq [t (into #{} search-tags)]
                          (db/tag-meme! {:reg-entry/address registry-entry :tag/name t})))))))))
 
-(defn param-change-constructed-event [_ {:keys [:args] :as event}]
-  (let [{:keys [:registry-entry :creator :version :deposit :challenge-period-end :db :key :value :timestamp]} args]
-    (promise-> (js/Promise.resolve (db/insert-registry-entry! {:reg-entry/address registry-entry
-                                                               :reg-entry/creator creator
-                                                               :reg-entry/version version
-                                                               :reg-entry/deposit (bn/number deposit)
-                                                               :reg-entry/challenge-period-end (bn/number challenge-period-end)
-                                                               :reg-entry/created-on timestamp}))
-               #(db/insert-or-replace-param-change! {:reg-entry/address registry-entry
-                                                     :param-change/db db
-                                                     :param-change/key key
-                                                     :param-change/value (bn/number value)
-                                                     :param-change/initial-value (:initial-param/value (db/get-initial-param key db))}))))
+(defn param-change-constructed-event [_ {:keys [:args]}]
+  (try-catch
+   (let [{:keys [:registry-entry :creator :version :deposit :challenge-period-end :db :key :value :timestamp :meta-hash]} args
+         hash (web3/to-ascii meta-hash)]
+      (add-registry-entry {:reg-entry/address registry-entry
+                           :reg-entry/creator creator
+                           :reg-entry/version version
+                           :reg-entry/created-on timestamp
+                           :reg-entry/deposit (bn/number deposit)
+                           :reg-entry/challenge-period-end (bn/number challenge-period-end)}
+                          timestamp)
+      (promise-> (look (server-utils/get-ipfs-meta @ipfs/ipfs hash))
```

Comment:
remove look

Reply:
Weird that was removed, I'm not seeing it anymore